primitives/staking/src/lib.rs

```diff
@@ -135,6 +135,32 @@ pub trait StakingInterface {
 		num_slashing_spans: u32,
 	) -> Result<bool, DispatchError>;
 
+	/// Get StakingLedger by controller account.
+	fn get_ledger(ctrl: &Self::AccountId) -> StakingLedger<Self>;
+
+	/// The ideal number of staking participants.
+	fn validator_count() -> u32;
+
+	/// Whether or not there is an ongoing election.
+	fn ongoing() -> bool;
+
+	/// Force a current staker to become completely unstaked, immediately.
+	///
+	/// The dispatch origin must be Root.
+	/// TODO: Do we need `num_slashing_spans` in the original method or should
+	/// we do Staking::<T>::slashing_spans(&stash).iter().count() within it?
+	/// Or should we introduce another method force_unstake_with_spans that will do it?
+	fn force_unstake(
+		origin: OriginFor<Self>,
+		stash: T::AccountId,
+		num_slashing_spans: u32,
+	) -> DispatchResult;
+
+	/// Checks whether an account `staker` has been exposed in an era.
+	/// TODO: shall we have this helper here or just expose ErasStakers
+	/// storage instead for the caller to do as they please?
+	fn is_exposed_in_era(staker: &T::AccountId, era: &EraIndex) -> bool;
```

Comment:
I think for now this API is fine.

Reply:
Although -- I take it back. We have `get_ledger`. a similar `get_exposure` would fit nicer next to it.

Reply:
It's quite a bit different, because we have multiple exposures. 
So do we just return all the exposures that match staker, era? Otherwise it's inefficient.

I think it's best to keep this api tbh, because we are iterating by prefix. Or otherwise `get_exposures` with era.

Reply:
This is actually fine -- see my comment about `get_ledger`.

Reply:
That's fine, we don't need eras anywhere else for now, so keeping this signature specific to the use-case sounds good and concise.